packages/itwinui-css/src/menu/menu.scss

```diff
@@ -25,9 +25,10 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
     border-radius: $iui-border-radius;
   }
 
-  &.iui-scroll {
+  &:where(.iui-scroll) {
```

Comment:
I think Jon did this to reduce the specificity of this selector so that it can be overridden more easily. He mentioned that `:where()` does not have any specificity at all, as also mentioned [here](https://css-tricks.com/almanac/selectors/w/where/). For example:

#### with `:where()`
![image](https://user-images.githubusercontent.com/45748283/184630595-7f62d82b-7a27-4459-b7e6-5d24c71e9e45.png)

#### without `:where()`
![image](https://user-images.githubusercontent.com/45748283/184630653-cdb4c997-a56a-4933-8e16-4dcaded91d80.png)

As VSCode is showing, the selector with `:where()` has a lower specificity and this is easier to override.

Reply:
I know what `:where` does, I'm saying it's not necessary here (especially with the `max-height` moved to the top) and is unrelated to this PR.

Our [contributing guide](https://github.com/iTwin/iTwinUI/blob/main/CONTRIBUTING.md?plain=1#L125) says: _avoid random "fixes" that are unrelated to the linked issue_.

Reply:
I removed `:where()`, but just to improve my understanding, why would it not be required here? Is lower specificity not good? Or is it good but we're not doing it here because a PR should only address the problem in the issue?

Reply:
0. As a rule of thumb, it's best to keep the changes as small as possible to avoid side effects. A hypothetical example: if the `iui-scroll` class was changed in the v1 branch, this change here would have led to an unnecessary merge conflict.
1. The reason to keep specificity low is to make it easy to override it later. I don't think it's necessary here, because `max-height` itself is set on the menu which already has 0-1-0. If the user wanted to override it, they will likely override `max-height`, not our CSS variable.
2. The `:where` selector is fairly new and doesn't work in older browsers (like IE11 which we unfortunately still support while iTwinUI-css is still pre-1.0).